jobs/integr8ly/ocp4/ocp4-install.yaml

```diff
@@ -1,23 +1,31 @@
 ---
 
 - job:
-    name: pds-ocp4-install
+    name: ocp4-install
     project-type: pipeline
-    description: "Installs Integreatly on RHPDS OCP4 cluster using Integreatly Operator"
+    description: "Installs RHMI on specified OCP4 cluster using Integreatly Operator"
     sandbox: false
     concurrent: true
     parameters:
         - string:
-            name: RHPDS_CITY
-            description: "City or customer for RHPDS cluster, for example qe-ocp4-[hash]"
+            name: CLUSTER_NAME
+            description: "City or customer for RHPDS cluster, for example qe-ocp4-[hash] or cluster name specified when creating it using openshift-install"
+        - string:
+            name: CLUSTER_DOMAIN
+            default: 'open.redhat.com'
+            description: "Cluster domain - open.redhat.com or example.opentlc.com for RHPDS clusters and intlyqe.com for clusters created using openshift-install"
+        - bool:
+            name: PDS_CLUSTER
+            default: true
+            description: "Indicates whether cluster was provisioned using RHPDS or openshift-install"
         - bool:
             name: SELF_SIGNED_CERTS
             default: true
             description: "Indicates whether cluster uses self signed certificates or not"
         - string:
             name: ADMIN
             default: 'opentlc-mgr'
-            description: "Admin username for the OCP4 cluster"
+            description: "Admin username for the OCP4 RHPDS cluster"
```

Comment:
[Minor] Say here this is only used in case of RHPDS. Say 'kubeadmin' will be used otherwise.

Reply:
I assume that it is default admin user for clusters created by our ocp4 pipeline